src/registrar.js

```diff
@@ -109,6 +111,11 @@ class Registrar {
    * @return {string} registrar identifier
    */
   register (topology) {
+    assert(
+      Topology.isTopology(topology) ||
+      MulticodecTopology.isMulticodecTopology(topology),
+      'topology must be an instance of interfaces/topology')
```

Comment:
`is-class` should handle the base class extension check, so we should just need to check that it is a Topology, `Topology.isTopology(topology)`

Reply:
Cool, that works :)